airbyte-config/specs/build.gradle

```diff
@@ -21,16 +21,6 @@ task generateSeedConnectorSpecs(type: JavaExec, dependsOn: compileJava) {
     args new File(project(":airbyte-config:init").projectDir, '/src/main/resources/seed')
 }
 
-task downloadCombinedConnectorCatalog(type: JavaExec, dependsOn: compileJava) {
```

Comment:
@pedroslopez Sanity check, we can delete all the download logic in connectors. Correct?

Reply:
Correct!